docs/en/credential-revocation.rst

```diff
@@ -0,0 +1,508 @@
+    :header-rows: 1
+
+    * - **Header**
+      - **Description**
+      - **Reference**
+    * - **typ**
+      - It MUST be set to ``status-assertion-request+jwt``.
+      - :rfc:`7516#section-4.1.1`.
+    * - **alg**
+      - A digital signature algorithm identifier such as per IANA "JSON Web Signature and Encryption Algorithms" registry. It MUST be one of the supported algorithms listed in the Section `Cryptographic Algorithms <algorithms.html>`_ and MUST NOT be set to ``none`` or any symmetric algorithm (MAC) identifier.
+      - :rfc:`7516#section-4.1.1`.
+
+
+.. list-table:: 
+    :widths: 20 60 20
+    :header-rows: 1
+
+    * - **Payload**
+      - **Description**
+      - **Reference**
+    * - **iss**
+      - Thumbprint of the JWK in the ``cnf`` parameter of the Wallet Attestation.
+      - :rfc:`9126` and :rfc:`7519`.
+    * - **aud**
+      - It MUST be set to the URL string of Credential Issuer Status Assertion endpoint.
+      - :rfc:`9126` and :rfc:`7519`.
+    * - **exp**
+      - UNIX Timestamp with the expiry time of the JWT. It MUST be greater than the value set for `iat`.
+      - :rfc:`9126` and :rfc:`7519`.
+    * - **iat**
+      - UNIX Timestamp with the time of JWT issuance.
+      - :rfc:`9126` and :rfc:`7519`.
+    * - **jti**
+      - Unique identifier for the proof of possession JWT. The value SHOULD be set using a *UUID v4* value according to [:rfc:`4122`].
+      - :rfc:`7519#section-4.1.7`.
+    * - **credential_hash**
+      - It MUST contain the hash value of a Digital Credential, derived by computing the base64url encoded hash of the Digital Credential.
+      - `OAUTH-STATUS-ASSERTION`_.
+    * - **credential_hash_alg**
+      - It MUST contain the Algorithm used for hashing the Digital Credential. The value SHOULD be set to `sha-256`.
+      - `OAUTH-STATUS-ASSERTION`_.
+
+
+Below, is given a non-normative example of a single *Status Assertion Request object* with decoded JWT headers and payload and without signature for better readability:
+
+.. _credential_pop_jwt_ex:
+.. code-block::
+
+    {
+      "alg": "ES256",
+      "typ": "status-assertion-request+jwt",
+    }
+    .
+    {
+      "iss": "0b434530-e151-4c40-98b7-74c75a5ef760",
+      "aud": "https://pid-provider.example.org/status",
+      "iat": 1698744039,
+      "exp": 1698744139, 
+      "jti": "6f204f7e-e453-4dfd-814e-9d155319408c",
+      "credential_hash": $Issuer-Signed-JWT-Hash,
+      "credential_hash_alg": "sha-256"
+    }
+
+
+HTTP Status Assertion Response
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In case of succesfully Status Assertion Request validation, the *Credential Issuer* MUST return an HTTP response with the status code set to *200 OK*. If the *Credential Issuer* is able to provide a valid Status Assertion for a requested Credential, the response MUST contains a Status Assertion object within a JSON Array. Otherwise, a Status Assertion Errors related to that Credential MUST be included in the Response JSON Array as an entry.
```

Comment:
```suggestion
In case of succesfully Status Assertion Request validation, the *Credential Issuer* MUST return an HTTP response with the status code set to *200 OK*. If the *Credential Issuer* is able to provide a valid Status Assertion for a requested Credential, the response MUST contain a Status Assertion object within a JSON Array. Otherwise, a Status Assertion Errors related to that Credential MUST be included in the Response JSON Array as an entry.
```